### PR TITLE
[wip] Test release 2.12.0

### PR DESCRIPTION
**Pull request by VaishnaviHire**
## Merge criteria:

- [ ] The commits are squashed in a cohesive manner and have meaningful messages.
- [ ] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [ ] The developer has manually tested the changes and verified that the changes work

### CODE REVIEW

File: get_all_manifests.sh [hunk 1]
```diff
@@ -6,20 +6,20 @@ GITHUB_URL="https://github.com/"
...
-    ["trainingoperator"]="opendatahub-io:training-operator:dev:manifests:trainingoperator"
+    ["codeflare"]="opendatahub-io:codeflare-operator:v1.4.1-odh:config:codeflare"
+    ["ray"]="opendatahub-io:kuberay:v1.1.0-odh:ray-operator/config:ray"
+    ["kueue"]="opendatahub-io:kueue:v0.6.2-odh:config:kueue"
+    ["data-science-pipelines-operator"]="opendatahub-io:data-science-pipelines-operator:v2.2.0:config:data-science-pipelines-operator"
+    ["odh-dashboard"]="opendatahub-io:odh-dashboard:v2.22.2-odh-release:manifests:dashboard"
+    ["kf-notebook-controller"]="opendatahub-io:kubeflow:v1.7.0-9:components/notebook-controller/config:odh-notebook-controller/kf-notebook-controller"
+    ["odh-notebook-controller"]="opendatahub-io:kubeflow:v1.7.0-9:components/odh-notebook-controller/config:odh-notebook-controller/odh-notebook-controller"
+    ["notebooks"]="opendatahub-io:notebooks:v1.18.0:manifests:notebooks"
+    ["trustyai"]="trustyai-explainability:trustyai-service-operator:release/1.20.0:config:trustyai-service-operator"
```
Comment (zdtsw) [inline get_all_manifests.sh:17]: should this be 
["trustyai"]="trustyai-explainability:trustyai-service-operator:v1.20.0:config:trustyai-service-operator"
?

Comment (VaishnaviHire) [reply get_all_manifests.sh:17]: Yes, but this is also a release branch, so I think it is okay for now. We can update it in next release

**Pull request closed without merging**